src/renderer/index.html

```diff
@@ -5,6 +5,6 @@
         <title><%= htmlWebpackPlugin.options.title %></title>
     </head>
     <body>
-        <div id="root" />
+        <div id="app" />
```

Comment:
We'd have to change a lot of different code to change that id. Since it doesn't really make a huge difference can we keep it the same for now? We could refactor this later.